src/mixins/test/wordWrap.test.js

```diff
@@ -0,0 +1,14 @@
+import wordWrap from '../wordWrap'
+
+describe('wordWrap', function() {
+
+  it('should accept other values', function() {
+    expect(wordWrap('break-all')).toMatchSnapshot()
+  })
+
+  it('should default wrap to break-word', function() {
+    expect(wordWrap()).toMatchSnapshot()
+  })
```

Comment:
Ah sorry, good call!

I just realized that we should have two examples per util, demonstrating usage with styles as objects and `styled-components` independently:

```JS
// Styles as object usage
const styles = {
  ...wordWrap(),
}

// styled-components usage
const Main = styled.main`
  ${wordWrap()}
`
```

It would be cool if this was globally togglable (i.e. choose if the whole site displays `styled-components` or styles as objects usage) somewhere so we don't show both all the time. What do you think, is that feasible?